### PR TITLE
Project Completion

### PR DESCRIPTION
**Pull request by Aaron-RN**
Hey I'm unable to get in contact with my coding partner so I just decided to finish off the HTML curriculum alone since only 2 more collab projects were left...

### CODE REVIEW

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hey @Aaron-RN  👋 
### well done 👏  and project looks good But why are you working alone?
#### Things to note
- padding can be used as `padding: 15px 10px;` that will take care of horizontal and vertical instead of declaring all four vales as you did in the main.css
- Take out the comments in the mark-up and I must commend your use of html tags like `nav` and `footer` instead of generic `div`
- please remove the css you added for debugging  purposes i suppose
- finally, we have some liniting problems with git, please look through and fix them.

**Request another review when this is done**

**Comment by Aaron-RN** [comment]
> # Hey @Aaron-RN 👋
> ### well done 👏 and project looks good But why are you working alone?
> #### Things to note
> * padding can be used as `padding: 15px 10px;` that will take care of horizontal and vertical instead of declaring all four vales as you did in the main.css
> * Take out the comments in the mark-up and I must commend your use of html tags like `nav` and `footer` instead of generic `div`
> * please remove the css you added for debugging  purposes i suppose
> * finally, we have some liniting problems with git, please look through and fix them.
> 
> **Request another review when this is done**

Can't get in contact with my coding partner...

**Comment by Aaron-RN** [comment]
> * Take out the comments in the mark-up and I must commend your use of html tags like `nav` and `footer` instead of generic `div`

What do you consider the mark-up?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-In this project, you’ll get the chance to build a responsive website. We’ll be cloning The Next Web, a tech-focused magazine which uses media queries to gracefully degrade their site as the window size is reduced.
+# (- PROJECT: Building with Responsive Design - )
+In this project, you’ll get the chance to build a responsive website. We’ll be cloning The Next Web, a tech-focused magazine which uses media queries to gracefully degrade their site as the window size is reduced. You can find the original project specification at: https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design
+
+## Authors: 
+### Aaron
+Aaron's Github Profile: https://github.com/Aaron-RN
+
+## Github Link
+Github Link: https://github.com/Aaron-RN/Building-With-Responsive-Design
```
Comment (bolah2009) [inline README.md:9]: #### `OPTIONAL`

Can you add screenshots different breakpoint of the responsive [clone](https://raw.githack.com/Aaron-RN/Building-With-Responsive-Design/82df751cb251b9f875b5f4d04067da4119c3a56f/navbar.html). I think that would give this `README` a more descriptive and professional look. :blush:

**Review by bolah2009 — APPROVED** [review]
# Nice job Team ... :clap: 

### The [clone](https://raw.githack.com/Aaron-RN/Building-With-Responsive-Design/82df751cb251b9f875b5f4d04067da4119c3a56f/index.html) looks good to me and responsive. :+1: 

For next (HTML/CSS) project, please follow `GitHub flow` [guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). What is missing here is a descriptive brance name instaed of `Development`.

#### [REQUIRED] Please I strongly recommend you do this before merging and taking note in next projects.
##### Make sure that repo looks professional
- Use meaningful commits messages. [Learn more here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
- Use meaningful names for  Pull Requests. See below.
- Add a short summary of changes in Pull Request description. See below.

#### :heavy_check_mark: This project is approved.
Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Review by bolah2009 — COMMENTED** [review]
# Rename PR

  rename PR

-----

# Update PR description

  update-PR-description-html

**Pull request merged**